src/test/integration/integration_tests.py

```diff
@@ -0,0 +1,52 @@
+import unittest, sys, json, math, operator
+from PIL import Image
+from api_request import *
+from configuration_load import *
+
+class gateway_tests(unittest.TestCase):
+
+    def test_verbs(self):
+        self.assertEqual(basic_request
+            (url, access_token, "get").status_code, 200)
+        self.assertEqual(basic_request
+            (url, access_token, "post").status_code, 405)
+        self.assertEqual(basic_request
+            (url, access_token, "put").status_code, 405)
+        self.assertEqual(basic_request
+            (url, access_token, "delete").status_code, 405)
+
+    def test_not_found(self):
+        bad_url = url + "012345678"
+        self.assertEqual(basic_request
+            (bad_url, access_token).status_code, 404)
+
+    def test_image(self):
```

Comment:
How about adding an image resizing test?
We have such as: osu_id_no_image in the config file. We could have another variable osu_id_valid_image that a developer could put their own id image to test and actually force the api to make a call to the backend?